plugins/memcached/lib_memcached.ecllib

```diff
@@ -37,4 +37,6 @@ export memcached := SERVICE : plugin('memcached'), namespace('MemCachedPlugin')
   BOOLEAN Exists(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partitionKey = '') : cpp,action,context,entrypoint='MExists';
   CONST VARSTRING KeyType(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partitionKey = '') : cpp,action,context,entrypoint='MKeyType'; //NOTE: calls get
   Clear(CONST VARSTRING options) : cpp,action,context,entrypoint='MClear';
+  Delete(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partionKey = '') : cpp,action,context,entrypoint='MDelete';
```

Comment:
I'm a little confused, the function is called 'delete' and is documented in the README (when it gets merged), so you want me to remove 'Del'?
Within the redis plugin, the documented function is called 'del' but the library also includes delete. I still stick by my thoughts on the redis flavour conforming with the actual redis command.
I have no objections to removing 'del' from the memcached library.